n6001/hardware/ofs_n6001/build/scripts/gen-asp-quartus-report.tcl

```diff
@@ -0,0 +1,144 @@
+# Copyright 2022 Intel Corporation
```

Comment:
We still need a script to generate the acl_quartus_report file, so I created a new one (from adjust_plls.tcl).